src/extractor/guidance/intersection_generator.cpp

```diff
@@ -158,7 +162,134 @@ Intersection IntersectionGenerator::getConnectedRoads(const NodeID from_node,
         intersection[self_u_turn].entry_allowed = true;
     }
 
-    return mergeSegregatedRoads(std::move(intersection));
+    return intersection;
+}
+
+bool IntersectionGenerator::CanMerge(const NodeID node_at_intersection,
```

Comment:
Great. Yeah I think the docs make the most sense where they are implemented, since this is all internal API (and you end up reading the implementation more often then not).